### DISCUSSION TITLE
Desenvolvedor vs RH

### DISCUSSION THREAD

**Discussion by gadielkalleb** [frontendbr/forum#663] [Discussão]
Hoje eu recebi uma ligação para uma vaga de trabalho, enviei meu curriculo com meu site github.
Busco minha primeira oportunidade na área tem alguns projetos que to fazendo tem uns freella. mas no curriculo deixei bem claro que registrado ou pj é meu primeiro emprego na área.

o Funcionario do rh queria que eu colocasse alguma  experiência, falei que tem meu github com meu repositorios, tem poucos e estou refatorando meus codigos antigos.

Meus professores do curso de desenvolvimento mobile ja me falaram que o github é otimo para potifolio e avaliação.
O mais complicado é vc explicar para alguem de rh oque é github
Como montar um curriculo se não tem uma experiência? ou explicar para alguem de rh como é o mundo de desenvolvimento.

pois fica dificil ser avaliado por alguem que não conhece o ramo e nem as ferramentas que estão no mercado

**Comment by marcusvcribeiro** [comment]
Sinceramente, esse é o ponto que mais tenho raiva.
Pessoas despreparadas fazendo entrevista com profissionais de TI.
Perguntam o que a equipe de TI usa na empresa e colocam no edital da vaga
um NINJAMOTHERFUCKER, ai quando você vai na entrevista a pessoa é de outra
área....

Ai você tem que ser aquele cara gente boa, para a pessoa do RH gostar de
você e passar para a outra etapa.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Livre
de vírus. www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>.
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

Em 15 de agosto de 2017 16:44, Gadiel Kalleb <notifications@github.com>
escreveu:

**Comment by rubenmarcus** [comment]
faz um curriculo coloca que você é front-end junior freelancer (manda seu linkedin ai, vou dar uns ups lá)

**Comment by rubenmarcus** [comment]
Na verdade tava vendo seu git e portfolio e estão bons, vc tem mais repositório que eu, Mas aqui http://kalleb.lrcompany.com.br/

Isso é de 2013. Na teoria não era pra vc ser junior. De 2013 a 2017 você deu uma parada?

Esse lance de rh, vixe, cara normal, eu ja fui em vaga que pediam MSN e ICQ na ficha de cadastro,ou que os caras faziam prova escrita a mão de código, com mais de 100 questões e redação (me senti no enem) , outras vagas que os caras pagavam mal e faziam seleções tipo code in the dark, o cara dava menos de 30minutos pra você montar um layout responsivo, sem emmet sem nada

Eu no momento to muito focado em freelas e remoto, mas to muito também em melhorar meu tempo para organização pessoal e estudos, e to passando a maioria das vagas, só quando é no exterior ou de empresas que eu gosto muito eu participo do processo seletivo, mais pra ver como é o processo, do que passar mesmo, porque no nivel que estou no momento acho dificil passar.

Mas tem que ter paciência, vagas ruins e RHS ruins é o que mais tem, isso é bom por que te prepara de certa forma para as boas também.

**Comment by gadielkalleb** [comment]
@rubenmarcus  aquele site foi o primeiro que eu fiz usando as aulas do curso em video ai parei e fui cursar matematica.
os repositorios aqui eu to refatorando principalmente os de javascript e ionic

mas meu site mesmo é o http://kallebdev.lrcompany.com.br
é ruim mesmo rh , ja fiz entrevista que a pessoal perguntou como eu criei esse tal de github...pode acreditar.

to atirando pra tudo que canto.

**Comment by Clayder** [comment]
Uma coisa que eu estou tentando fazer é colocar meus projetos no meu servidor e os que não da para colocar no servidor eu faço um vídeo mostrando as funcionalidades, aí coloco os links no currículo, na parte de experiências.

**Comment by marlon-net** [comment]
RH despreparado. Empresa que não entende ainda que esse tipo de posição ou não pode começar com entrevista com RH ou o RH só pode fazer perguntas não técnicas só pra saber se o candidato não é um serial killer ou algo do gênero.... Foi só isso. 

-
Marlon Dias

**Comment by rubenmarcus** [comment]
pra descontrair

[logo]: http://i.imgur.com/9O0zyuN.jpg ""

![alt text][logo]

+ aqui http://tidiota.tumblr.com/

**Comment by gadielkalleb** [comment]
Kkkkkkk

**Comment by deppbrazil** [comment]
@gadielkalleb aqui vai alguns pontos no qual você pode melhorar...

Entenda que o RH em algumas empresas não vão entender mesmo o que é GitHub, em muitos caso nem devem, isso vai acontecer até mesmo em empresas grandes, normal... 

Muitas empresas tem seus próprios HeadHunters para já mandarem para o RH o candidato perfeito para a vaga... vai muito da cultura da empresa...

Em questão da prova manual isso é muito normal em GPTW a [Ci&T](http://www.ciandt.com/home) por exemplo faz muito isso, pois apenas para avaliar se vc tem noção do html, css e js fora do pc... isso é muito bom para sair do automático dos editores de texto com plugins de autocomplite... fique tranquilo eles não vão avaliar o erro no script caso tenha esquecido de fechar chaves..rsrs é apenas para saber se vc tem noção....

Como vc não tem experiência comprovada coloque no currículo suas Skills, seu objetivo no mundo Dev. sua personalidade, os curso que fez ou ainda faz etc, redes sociais como LinkedIn, endereço de seus repositório, site e portfólio... Lembrando que o currículo precisa ser claro e objetivo, dados como rua onde vc mora, RG, jamais nos dias de hj rsrs... Lembre-se o RH se foca em palavras chaves do seu currículo... 

A comunicação no dia da entrevista é o ponto principal, ser comunicativo é um baita diferencial, + o mkt pessoal tb conta, saiba se vender na hora certa... isso vai fazer com que o RH goste do seu perfil e ai sim te encaminhe para uma segunda etapa com o gestor da vaga, agora uma entrevista mais técnica que vc pode falar de GitHub com ele..rsrs 

Bom se pretende partir para o caminho de Front melhore seus repositórios em questão de documentação, semântica, organização do código, comentários, msg de commits etc... 
**(vi mtas msg de commits seus com links https e nomes muito grandes e comentários meio poluídos visualmente)** 

Tente ser direto ao ponto, tipo: `BugFixes + New Features` `Add (nomeDoArq.)` etc tente fazer algo padrão e coerente com o conteúdo do commit, pois se vc trabalhar em projetos com mais desenvolvedores lembre que eles tb verão seus commits.

O ponto principal que os gestores, HeadHunters, empresas voltadas ao mundo do Front verão no seu perfil do GitHub é seu estilo de código e se vc sabe trabalhar com repositórios e versionar com git, pois a maioria das empresas usam muito isso, organização e padrão ajuda tb ainda mais se a empresa trabalha com Scrum... noção de UX/UI tb é um diferencial que empresas procuram mto para a área de Front.

Enfim procure Devs mais experientes no GitHub e tire algumas ideias em seus repositórios, ou até mesmo em projetos grandes... Uma boa dica para documentação de projetos é o repo do [CAKE PHP](https://github.com/cakephp/cakephp)

=)

**Comment by WandersonAlves** [comment]
Cara, não sei se contribui direto com a issue, mas o que aconteceu comigo a 2 meses atrás foi o seguinte:

Queria MUITO sair de onde trabalhava, então fiz algo que eu já queria a eras que foi **deixar meu CV bonito**.

Quando falo bonito é BONITO mesmo, e a partir do momento que fiz isso, em vez de ser a pessoa que procura emprego, eu era **procurado** para trabalhar. Claro, tenho um git razoavelmente bom e um linkedin melhor ainda. Mas o máximo que você puder fazer pra destacar o seu cv, deixar as coisas recheadas, faça!

E inclusive, cai na seleção de um agencia e do nada os caras me pedem uma redação de 30 linhas com tema aberto e uma outra prova sobre CSS, JS e HTML5 (perguntas técnicas, nada de código). Preciso nem falar que odiei rsrs

CV Antes: https://drive.google.com/file/d/0B1pWfEOl_3e9dDVieUF2MFZBYk0/view?usp=sharing
CV Depois: https://drive.google.com/file/d/0B1pWfEOl_3e9TmFRY1JfaHVHdjQ/view?usp=sharing

**Comment by patricknerbas** [comment]
Geralmente quem analisa os currículos é a equipe técnica da empresa, o RH faz esse meio de campo entre o candidato e a parte técnica. Se o RH está analisando os currículos, isso na minha opinião o processo da empresa me parece um pouco destorcido.

Por exemplo, na empresa onde trabalho, o RH inicialmente fez umas perguntas básicas, como questão salarial, deslocamento, etc - isso por telefone. Depois me encaminhou uma prova para avaliar o conhecimento técnico, então, finalmente uma entrevista presencial na empresa com o Gestor do Projeto, Basicamente, foi esse o fluxo do meu processo de contratação.

**Comment by rubenmarcus** [comment]
@WandersonAlves  Po legal cara, pensei que o aspecto visual impactava mais pra Designers do que pra Devs. Legal ler seu feedback, eu vou fazer algo assim também.

Uma pergunta: Você saiu da Accenture para ir pra uma empresa menor? Geralmente os profissionais não fazem o contrario?

Eu trabalhei de 2012 a 2015 em Agências e Startups e ai resolvi que precisava ir pra uma empresa maior, ecommerce de preferência, ai fui pra Centauro. Depois para Printi. Ai sai da Printi, e agora estou só com freelas, e tentando vagas remoto. To num processo aqui da Crossover, mas não sou tão habilidoso assim em JavaScript e esses frameworks MVVM novos. 

Digo eu sei o básico do JS e sei me virar aprender o que preciso pra desenvolver, mas ainda não me sinto seguro pra falar que domino. Na verdade eu to meio que passando uma crise e não sei se quero continuar sendo Front, se vou pra caminho de UX ( Mesmo não tendo formação nem em Ciencias da Computação e nem em Design)

E ai parei agora por uns 6 meses pra estudar as possibilidades. Quero ver se retomo o Curso de UX que estava fazendo no Coursera e parei pelos freelas, e também retomo cursos de JS ...

Eu gosto MESMO de HTML + CSS ahahahah mas ai fica complicado não dá pra trabalhar só com isso.

**Comment by WandersonAlves** [comment]
@rubenmarcus Meu perfil é bem dev/ui/ux MESMO, desde antes de dev que eu já brincava de projetar telas e tal. Minha motivação pra fazer um CV bonitinho foi a de que a pessoa do RH quando tiver olhando os CV's, vai passar por um altamente fora do padrão e vai se interessar na hora. Minha jogada foi essa ai e deu certo.

> Uma pergunta: Você saiu da Accenture para ir pra uma empresa menor? Geralmente os profissionais não fazem o contrario?

Quanto a isso eu escolhi sair por dois motivos:

- Desvalorização do meu trabalho e skills
- Qualidade de vida

Não é toda empresa nem muito menos todo projeto dessa empresa que te deixa "mal" assim, porém comigo tive esse azar e vindo pra um lugar menor, tenho mais tempo pra viver e tocar projetos paralelos, estudar, etc etc. Basicamente sai pela qualidade de vida (e pelo salário que aumentou consideravelmente)

E na tua pegada cara, vai de PS, Adobe XD e similares, vai fazendo por tu mesmo que uma hora chega.
Aqui em Recife a área de UX/UI é bastante ruim, mas mesmo assim vou tocando essas áreas pra ser mais "completo"

Aprende um framework rápido de aprender (Vue por exemplo) e começa a fazer tuas interfaces nele, é divertido pra porra!

**Comment by rubenmarcus** [comment]
@WandersonAlves  Então mano eu to numa sinuca de bico. Nas empresas que trabalhei se utilizava Angular JS , e muita coisa mesmo feito em Angular, mas minha posição como front, exigia mais que eu fizesse o recorte e implementação e pouco da funcionalidade. Então eu mexi muito pouco com a parte lógica, mexia mais nas chamadas e interações de tela.

Quando sai de lá meu gestor falou pra eu aprender React. Mas React eu não to achando tão fácil também não, você pega uns testes e exemplos de React, ai tem que fazer a interação com Express, ai Webpack, ai monte de coisas, eu vi que seu curriculo você tem bastante experiência no lance de JS e essa parte que está faltando pra mim mesmo, vi ali lance de CI que acho muito louco, quero aprender isso tudo também mano, po valeu pelo depoimento, da hora ver isso

Eu acho que esses anos todos da minha carreira, faltou um pouco de comprometimento meu com a educação minha quanto profissional, eu gosto de ler muito e ver o que os outros estão fazendo, as vezes testar uma coisa ou outra, mas nunca fiz projetos muito grandes.. Um plano meu agora é usar esses meses restantes até 2018 e construir um app, com conteúdo colaborativo.

**Comment by WandersonAlves** [comment]
@rubenmarcus Eu fiquei 2 anos e meio trabalhando com AngularJS, no inicio achava o máximo. Com o passar do tempo vi que tava ficando cada vez mais complicado realizar coisas simples com ele (devido a complexidade dos projetos que atuava) e então por fora resolvi estudar outras coisas.

Tem amigos meus que ainda continuam trabalhando com AngularJS 1.x e querem sair mas não conseguem pois não vêem mais tempo pra fazer outras coisas. Um mesmo já disse que está perdendo o gosto em programar devido a isso :/

Estudar por fora, ler sobre coisas que não se usa no seu trabalho não te fazem ficar pra trás, e sim pra frente, e ai algum momento alguém vai dizer "po, a performance aqui tá baixa, a manutenção tá péssima" e você fala "se tivessem me ouvido lá no inicio, não tavam assim" (aconteceu comigo kkk). E é nesse momento que você percebe que é hora de seguir.

(acho que já fugimos do tema da issue kkkkkkkkkk)
